validation/src/main/scala/hmda/validation/engine/lar/validity/LarValidityEngine.scala

```diff
@@ -3,27 +3,21 @@ package hmda.validation.engine.lar.validity
 import hmda.model.fi.lar.LoanApplicationRegister
 import hmda.validation.api.ValidationApi
 import hmda.validation.engine.lar.LarCommonEngine
+import hmda.validation.rules.EditCheck
 import hmda.validation.rules.lar.validity.{ V262, V220, V225 }
 
 trait LarValidityEngine extends LarCommonEngine with ValidationApi {
-  private def v220(lar: LoanApplicationRegister): LarValidation = {
-    convertResult(lar, V220(lar.loan), "V220")
-  }
-
-  private def v225(lar: LoanApplicationRegister): LarValidation = {
-    convertResult(lar, V225(lar), "V225")
-  }
 
-  private def v262(lar: LoanApplicationRegister): LarValidation = {
-    convertResult(lar, V262(lar), "V262")
+  private def doCheck(editCheck: EditCheck[LoanApplicationRegister], lar: LoanApplicationRegister): LarValidation = {
```

Comment:
Could we change this to `check`? I think it reads better and is more in line with the rest of the codebase.